src/cli/commands/lint.ts

```diff
@@ -150,6 +151,24 @@ async function lint(name: string, flags: ILintConfig, command: Lint, rules?: Rul
     mergeKeys: true,
   });
   const spectral = new Spectral({ resolver: httpAndFileResolver });
+  spectral.registerFormat('oas-2', document => {
+    if (isOpenApiv2_0(document)) {
+      command.log('OpenAPI 2.0 (Swagger) detected');
+      return true;
+    }
+
+    return false;
+  });
+
+  spectral.registerFormat('oas-3', document => {
+    if (isOpenApiv3(document)) {
```

Comment:
It’d more important for v3 to have the _0 because it’s gonna get v3.1 at some point. Should we leave them as OpenApiv2 and OpenApiV3 then just add _1 when it comes out? If so we can probably ignore the above comments on format names.

Reply:
Having `oas2`, `oas3` and potentially `oas3.1` seems like a reasonable choice.
Are we sticking to this?